package-lock.json

```diff
@@ -1412,11 +1412,27 @@
       "integrity": "sha512-EUCPEkaRPvmHjWAAZkWMT7JDzpw7FKB00WTISaiXsbNOd5hCHg77XLA8sLYLFDo1zepYLo2w7GstN8YBqRXZfA==",
       "dev": true,
       "requires": {
-        "convert-source-map": "1.8.0",
-        "istanbul-lib-instrument": "4.0.3",
-        "loader-utils": "2.0.0",
-        "merge-source-map": "1.1.0",
-        "schema-utils": "2.7.1"
+        "convert-source-map": "^1.7.0",
+        "istanbul-lib-instrument": "^4.0.3",
+        "loader-utils": "^2.0.0",
+        "merge-source-map": "^1.1.0",
+        "schema-utils": "^2.7.0"
+      }
+    },
+    "@magnite/fledge.polyfill": {
+      "version": "0.3.2",
+      "resolved": "https://registry.npmjs.org/@magnite/fledge.polyfill/-/fledge.polyfill-0.3.2.tgz",
```

Comment:
appears to have some redundant functionality to https://magniteengineering.github.io/fledge.polyfill/scripts/frame/esm/index.js  ; both contain a runAdAuction method

Reply:
they are the same; the URL you've provided is the hosted version of this for testing purposes

Reply:
there are hundreds of changes that have nothing to do with this pr

Reply:
I am not clear based on this if your goal is a merge with this pr or something else. We're not going to merge code you say is redundant.

Reply:
I'm not sure I understand your question or concern here